apps/google-analytics-4/frontend/src/components/common/GoogleAnalyticsIcon.tsx

```diff
@@ -0,0 +1,52 @@
+import { css } from 'emotion';
```

Comment:
Common Pattern Comment: There could be multiple usages that don't really belong to a "location" IFrame so put those components in the /common directory (Credit: Lillian Bitner)